scripts/testdata/hello_world_compat_golden/.dockerignore

```diff
@@ -0,0 +1,19 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
This appears to have been broken by my change, although it's quite a headscratcher.

In this branch the .dockerignore file is generated w/o the Copyright, whereas it appears to be generated with it prior.

This break occurs for both hello_world and the compat equivalent tests.

Reply:
The python-compat .dockerignore doesn't have the copyright statement, while the python .dockerignore does, for no particularly good reason :).

Create a `data/dockerignore.python_compat` file with this:

```
.dockerignore
Dockerfile
.git
.hg
.svn
```

Can you also move the COMPAT_DOCKERFILE contents into a `data/Dockerfile.python_compat` file for symmetry?